Cargo.toml

```diff
@@ -56,6 +56,11 @@ features = ["std"]
 [dev-dependencies.serde_test]
 version = "1.0"
 
+[dev-dependencies.quickcheck]
```

Comment:
Why is this a dev-dependency? As a regular dependency, it can be optional, and thus we can choose to support it only for sufficiently new compilers. It also needs to be that way for others to use it, per #5, rather than just `cfg(test)`.

So we could use 0.8 that way. The caveat is that it becomes a public dependency, so we can't bump semver on it without bumping semver ourselves too.

Reply:
I didn't realize how optional dependencies work. I'll switch that over now.